content/en/docs/v3.5/op-guide/kubernetes.md

```diff
@@ -0,0 +1,438 @@
+        ## Here is what a single parameterized peer looks like:
+        ##   etcd-0=$(URI_SCHEME)://etcd-0.$(SERVICE_NAME):2380
+        - --initial-cluster=etcd-0=$(URI_SCHEME)://etcd-0.$(SERVICE_NAME):2380,etcd-1=$(URI_SCHEME)://etcd-1.$(SERVICE_NAME):2380,etcd-2=$(URI_SCHEME)://etcd-2.$(SERVICE_NAME):2380
+        ##
+        ## The peer urls flag should be fine as-is.
+        - --initial-advertise-peer-urls=$(URI_SCHEME)://$(HOSTNAME).$(SERVICE_NAME):2380
+        ##
+        ## This avoids probe failure if you opt to configure TLS.
+        - --listen-metrics-urls=http://0.0.0.0:8080
+        ##
+        ## These are some configurations you may want to consider enabling, but
+        ## should look into further to identify what settings are best for you.
+        # - --auto-compaction-mode=periodic
+        # - --auto-compaction-retention=10m
+        ##
+        ## TLS client configuration for etcd, reusing the etcdctl env vars.
+        # - --client-cert-auth
+        # - --trusted-ca-file=$(ETCDCTL_CACERT)
+        # - --cert-file=$(ETCDCTL_CERT)
+        # - --key-file=$(ETCDCTL_KEY)
+        ##
+        ## TLS server configuration for etcdctl in the container.
+        ## These files paths are part of the "etcd-server-certs" volume mount.
+        # - --peer-client-cert-auth
+        # - --peer-trusted-ca-file=/etc/etcd/certs/server/ca.crt
+        # - --peer-cert-file=/etc/etcd/certs/server/tls.crt
+        # - --peer-key-file=/etc/etcd/certs/server/tls.key
+        ##
+        ## This is the mount configuration.
+        volumeMounts:
+        - name: etcd-data
+          mountPath: /data
+        ##
+        ## TLS client configuration for etcdctl
+        # - name: etcd-client-tls
+        #   mountPath: "/etc/etcd/certs/client"
+        #   readOnly: true
+        ##
+        ## TLS server configuration
+        # - name: etcd-server-tls
+        #   mountPath: "/etc/etcd/certs/server"
+        #   readOnly: true
+      volumes:
+      ##
+      ## TLS client configuration
+      # - name: etcd-client-tls
+      #   secret:
+      #     secretName: etcd-client-tls
+      #     optional: false
+      ##
+      ## TLS server configuration
+      # - name: etcd-server-tls
+      #   secret:
+      #     secretName: etcd-server-tls
+      #     optional: false
+  ##
+  ## This StatefulSet will uses the volumeClaimTemplate field to create a PVC in
+  ## the cluster for each replica. These PVCs can not be easily resized later.
+  volumeClaimTemplates:
+  - metadata:
+      name: etcd-data
+    spec:
+      accessModes: ["ReadWriteOnce"]
+      ##
+      ## In some clusters, it is necessary to explicitly set the storage class.
+      ## This example will end up using the default storage class.
+      # storageClassName: ""
+      resources:
+        requests:
+          storage: 1Gi
+```
+
+# Generating Certificates
+In this section, we use [helm](https://helm.sh) to install an operator called [cert-manager](https://cert-manager.io/).
```

Comment:
I think Helm should be capitalized.

Reply:
I disagree on this one. Yes, normally names should be capitalized, but contextually the binary is lower-cased.

If this is a sticking point, I wont be offended if someone else makes this change.

Reply:
I see your point. I opened https://helm.sh to check how they use their name, and the result was that it's capitalized. In the case of referring to the binary, another option would be to do [`helm`](...). But, to be honest, I don't have a big inclination for either. Again, I just checked their site :)
Thanks for addressing this.